plugin-server/src/main/ingestion-queues/session-recording/blob-ingester/offset-manager.ts

```diff
@@ -94,9 +95,12 @@ export class OffsetManager {
 
         if (!inFlightOffsets) {
             gaugeOffsetRemovalImpossible.inc()
-            status.warn('💾', `offset_manager - no inflight offsets found to remove`, { partition })
```

Comment:
Couldn't this happen if the flush callback is triggered after the partition is revoked? Do we handle that case?

Reply:
OK, changed this around...

The question becomes

1) are we not removing session managers correctly on revoke of a partition
2) are we not tracking offsets correctly on assign

(or hopefully not 3) both)

So, explicitly track revoked and re-assigned partitions in the offset manager so we can distinguish between them